docs/DeveloperGuide.md

```diff
@@ -0,0 +1,115 @@
+# RepoSense - Developer Guide
+Thank you for contributing to RepoSense!
+- [Setting Up](#setting-up)
```

Comment:
Setting Up -> Setting up

The header that follows after the table of content does not capitalize U